app/management/commands/rundev.py

```diff
@@ -16,6 +73,9 @@ def handle(self, *args, **options):
             shell=True,
         )
 
+        check_env()
```

Comment:
should we have this return a true or false value and auto-quit if we are missing a known property?